jubilant/_juju.py

```diff
@@ -703,6 +703,36 @@ def run(
             if params_file is not None:
                 os.remove(params_file.name)
 
+    def scp(
+        self,
+        source: str,
+        destination: str,
+        *,
+        container: str | None = None,
+        host_key_checks: bool = True,
+        scp_options: Iterable[str] = (),
+    ) -> None:
+        """Securely transfer files within a model.
+
+        Args:
+            source: Source of file, in format ``[[<user>@]<target>:]<path>``.
+            destination: Destination for file, in format ``[<user>@]<target>:[<path>]``.
```

Comment:
Maybe a small typo here?
```suggestion
            destination: Destination for file, in format ``[<user>@]<target>[:<path>]``.
```

Reply:
Good call, fixed.